src/Toolbox/Toolbox.ts

```diff
@@ -0,0 +1,78 @@
+
+import { ITransmuteFramework } from '../TransmuteFramework';
+
+import * as util from 'ethereumjs-util';
+
+var bip39 = require("bip39");
+var hdkey = require('ethereumjs-wallet/hdkey');
+
+export class Toolbox {
+
+  constructor(
+    public framework: ITransmuteFramework,
+  ) {
+
+  }
+
+  public sign = (address, message) => {
+    return new Promise((resolve, reject) => {
+        // console.log(message.length)
+        // THIS IS LAME... testrpc... I hate you....
+        this.framework.web3.eth.sign(address, message, (err, signature) => {
+            if (err) {
```

Comment:
the way eth.sign works depends on the provider, node and wallet... sometimes only sha3 length messages can be signed.